src/domains/sign/cosmos_issuer.tf

```diff
@@ -79,3 +79,18 @@ module "cosmosdb_sql_container_issuer-issuers-by-vat-number" {
 
   default_ttl = var.io_sign_database_issuer.issuers.ttl
 }
+
+module "cosmosdb_sql_container_issuer-issuers-ip-whitelist" {
+  source              = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cosmosdb_sql_container?ref=v4.1.11"
+  name                = "issuers-ip-whitelist"
+  resource_group_name = azurerm_resource_group.data_rg.name
+  account_name        = module.cosmosdb_account.name
+  database_name       = module.cosmosdb_sql_database_issuer.name
+  partition_key_path  = "/id"
+
+  autoscale_settings = {
+    max_throughput = var.io_sign_database_issuer.uploads.max_throughput
+  }
+
+  default_ttl = var.io_sign_database_issuer.issuers.ttl
+}
```

Comment:
we shouldn't be using the same database used by other microservices, APIM should write to its own dedicated database

cc @manuelcoppotelli

Reply:
The key is read-only!

Reply:
I do not agree. Here the APIM only has read permission on the database and I don't see the reason to use another one!

Reply:
@grausof @lucacavallaro I think this is not the right place to take this kind of decisions... our `IO-SIGN-RFC-0024` was made with this scope and we are open to review it in case the outcome would not be good for us.